src/main/java/de/epiceric/shopchest/ShopCommand.java

```diff
@@ -421,65 +423,77 @@ private void create(String[] args, ShopType shopType, Player p) {
                 itemStack = new ItemStack(mat, 1);
             }
 
-            if (itemStack.getType().equals(Utils.getPreferredItemInHand(p).getType()) && itemStack.getDurability() == Utils.getPreferredItemInHand(p).getDurability()) {
-                if (buyEnabled) {
-                    if ((buyPrice > amount * maxPrice) && (buyPrice > 0)) {
-                        p.sendMessage(LanguageUtils.getMessage(LocalizedMessage.Message.BUY_PRICE_TOO_HIGH, new LocalizedMessage.ReplacedRegex(Regex.MAX_PRICE, String.valueOf(amount * maxPrice))));
-                        return;
+            if (itemStack.getType().equals(Utils.getPreferredItemInHand(player).getType()) && itemStack.getDurability() == Utils.getPreferredItemInHand(player).getDurability()) {
+                if (plugin.getShopChestConfig().maximum_buy_prices) {
```

Comment:
Then maybe it would be better to create another ``maximum-prices`` list for selling